### PR TITLE
Mobile menu

### PR DESCRIPTION
**Pull request by BlesslinJerishR**
## In this Branch

- Add script.js
- Add Mobile Menu
- Add X option
- Fix Linter errors

### CODE REVIEW

**Comment by didierganthier** [comment]
Hi @BlesslinJerishR ,

While you put in a lot of hard work in this project, unfortunately, I cannot proceed with the review of your code.



## Invalid Code Review Request 🚫

- You have submitted a project with your README missing the `Getting Started` section and it's subsections 👇🏽 

Screenshot 2023-04-10 at 10 25 17

Make sure to add them both in the `Table of Contents` and on the body and submit again for a review.

Your Code Review Request will be marked as invalid in your Dashboard, but don't worry you can always try again! 💪

Cheers and Happy coding!👏👏👏

If you have any questions or need clarification on anything, feel free to leave a comment in the pull request thread and don't forget to tag me so I'll receive the notification.

**Please note**
- To avoid confusion and delays, please do not open a new pull request for re-reviews. Use the same pull request submitted for the first review, unless otherwise instructed.

--------

_Invalid Code Review Request **does not count** towards the code reviews limit._

**Comment by prolajumokeoni** [comment]
Hi @Team

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a pair-programming project, that does not have commit from your partner, and  that does not have the javascript linters also setup, for this project both the [JavaScript](https://github.com/microverseinc/linters-config/tree/master/javascript) and [HTML&CSS](https://github.com/microverseinc/linters-config/tree/master/html-css) linters should be configured and  kindly check the work[ mode submission](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/7_mobile_menu.md#work-and-submission-mode) requirements 👍🏿

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: index.html [hunk 1]
```diff
@@ -14,8 +14,34 @@
...
+          id="x"
+          src="https://img.icons8.com/material-outlined/24/null/delete-sign.png"
+        />
+        <ul>
+          <li>
+            <a onclick="clickX()" href="#portfolio" class="clr-grey"
+              >Portfolio</a
+            >
+          </li>
+          <li>
```
Comment (emmiiorji) [inline index.html:30]: Kindly ensure that the design of the menu button is as close to the design as possible and looks same on mobile sizes.

Your design:
image

Figma design:
image

Notice how the dropdown covers the width of the screen.

File: index.html [hunk 2]
```diff
@@ -14,8 +14,34 @@
   </head>
   <body>
     <section id="content-1">
+      <div id="ham-menu">
+        <!-- Becky Add X Image -->
+        <img
+          onclick="clickX()"
+          id="x"
+          src="https://img.icons8.com/material-outlined/24/null/delete-sign.png"
+        />
```
Comment (emmiiorji) [inline index.html:23]: Kindly include some margin to the harmburger image. The menu takes the position of the close button when the menu is hidden.

Your design:
image

Figma design:
image

**Review by emmiiorji — CHANGES_REQUESTED** [review]
Hi Team, 
# Status: Changes Required ♻️

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

<div align='center'>

</div>


## Good Highlights✔️

- The menu button works
- Menu closes on click of the close button
- Menu items link to correct sections of the page


## Required Changes ♻️
_Check the comments under the review._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Olamarx — APPROVED** [review]
Hi @BlesslinJerishR, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- Linters passed ✔️ 
- Readme file ✔️ 

## Optional suggestions
- [OPTIONAL] I would  encourage you to add the link to be cloned in your readme file as it is missing and it will make it easier for people to know where or the exact link to clone in the repo.
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**